react/ui-components/src/hoc/FormComposerV2.js

```
@@ -373,66 +393,109 @@ export const FormComposer = (props) => {
-      )}
-      {showErrorToast && <Toast type={"error"} label={t("ES_COMMON_PLEASE_ENTER_ALL_MANDATORY_FIELDS")} isDleteBtn={true} onClose={closeToast} />}
-      {customToast && <Toast type={customToast?.type} label={t(customToast?.label)} isDleteBtn={true} onClose={closeToast} />}
-    </form>
-  );
+              })
+            ) : (
+              <Card style={getCardStyles()} noCardStyle={props.noCardStyle}>
+                {props?.config?.map((section, index, array) => {
+                  return section.navLink ? (
+                    <>
+                      <div style={section.navLink !== activeLink ? { display: "none" } : {}}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)